lib/pymedphys/tests/dicom/test_coords.py

```diff
@@ -97,23 +96,24 @@ def test_extract_dicom_patient_xyz():
 def test_coords_in_datasets_are_equal():
     bits_allocated = 32
 
-    test_dicom_dict = {
-        "ImagePositionPatient": [-1.0, -1.0, -1.0],
-        "ImageOrientationPatient": [1, 0, 0, 0, 1, 0],
-        "BitsAllocated": bits_allocated,
-        "BitsStored": bits_allocated,
-        "Rows": 3,
-        "Columns": 3,
-        "PixelRepresentation": 0,
-        "SamplesPerPixel": 1,
-        "PhotometricInterpretation": "MONOCHROME2",
-        "PixelSpacing": [1.0, 1.0],
-        "GridFrameOffsetVector": [0.0, 1.0, 2.0],
-        "PixelData": np.ones((3, 3, 3)).tobytes(),
-    }
+    ds1 = create.dicom_dataset_from_dict(
+        {
+            "ImagePositionPatient": [-1.0, -1.0, -1.0],
+            "ImageOrientationPatient": [1, 0, 0, 0, 1, 0],
+            "BitsAllocated": bits_allocated,
+            "BitsStored": bits_allocated,
+            "Rows": 3,
+            "Columns": 3,
+            "PixelRepresentation": 0,
+            "SamplesPerPixel": 1,
+            "PhotometricInterpretation": "MONOCHROME2",
+            "PixelSpacing": [1.0, 1.0],
+            "GridFrameOffsetVector": [0.0, 1.0, 2.0],
+            "PixelData": np.ones((3, 3, 3)).tobytes(),
+        }
+    )
 
-    ds1 = create.dicom_dataset_from_dict(test_dicom_dict)
-    ds1.fix_meta_info(enforce_standard=False)
+    compat.ensure_transfer_syntax(ds1)
```

Comment:
Worth just calling this inside dcm from dict?  (No need to change for this PR)

Reply:
Yep, good idea!